db.js

```diff
@@ -0,0 +1,44 @@
+const mysql = require('mysql2/promise');
+
+let connection = undefined
+ mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    database: 'anomalyDB',
+    password: 'password',
+  }).then((conn) => {
+    connection = conn;
+  })
+
+const getLastIPAddress = async (userId) => {
+    try {
+        const [results, fields] = await connection.execute(
+            'SELECT * FROM `IPHistory` WHERE `UserID` = ?',
+            [userId]
+          );
+            return results;
+    }   catch (error) {
+        console.log('error is ', error);
+    }
+}
+
+const insertLastIPAddress = async (userId, newIpAddress, lat, long) => {
+    const [results, fields] = await connection.execute(
+        'INSERT INTO `IPHistory` (UserID, IPAddress, LatLong) VALUES (?,?,Point(?,?))',
```

Comment:
where does the timestamp get inserted?

Reply:
Also, whats the db schema? Whats the primary key?

Reply:
Timestamp gets insered from the DB itself as default value. Here is the schema:

```
CREATE TABLE LastIPAddress (
    IPAddress varchar(48),
    UserID varchar(36),
    TenantID varchar(36),
    LatLong POINT,
    `timestamp` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    PRIMARY KEY (UserID)
) ENGINE=InnoDB;
```